acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/timeouts/TimeoutAcceptanceTest.java

```diff
@@ -38,14 +39,18 @@
 import org.web3j.utils.Convert.Unit;
 
 public class TimeoutAcceptanceTest {
+  private static final int UNUSED_PORT_A = 7007;
```

Comment:
Really we don't / there's no guarantee ...but according to https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers those are not reserved/used ports.

Reply:
Could the ethsigner when getting allocated a dynamic port potentially take 7007 by accident?
Could this be changed to deliberately be one-larger/smaller than the port allocated to Pantheon?

Reply:
Depends on how the underlying OS assigns ports.

The dynamically assigned ports I've read from the output locally & on Jenkins have been in the 6/5 digits

Reply:
> ANA specifies the range 49152 to 65535 be used for dynamic (AKA private) ports

Read more: https://blog.fpmurphy.com/2015/02/ip-dynamic-port-range.html#ixzz5nwVyKcl7
https://blog.fpmurphy.com/2015/02/ip-dynamic-port-range.html

Reply:
@rain-on in theory 7007 should be fine

Reply:
Instead of hard coding it, why not just allocate a free port and bind it and not do anything with the bound socket. So essentially creating a non-responsive server by using ServerSocket(0) to allocate and bind and then not use it, that way the server won't respond and nobody else can grab the allocated port.

Reply:
Done